test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1391,6 +1395,9 @@ protected static boolean isXPackTemplate(String name) {
         if (name.startsWith(".transform-")) {
             return true;
         }
+        if (name.startsWith(".deprecation-")) {
```

Comment:
This is for: `.deprecation-indexing-mappings`,  `.deprecation-indexing-settings` and `.deprecation-indexing-template` templates.